README.md

```diff
@@ -64,7 +64,35 @@ Then you will see something like this:
 To get ruff automatically running install the following Plugin:
 - [VSCode - Ruff Plugin](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff)
 
-### Continuous Delivery
+
+## Testing
+
+A test suite of a typical Python package consists of unit, integration, and documentation tests
+(i.e. the code snippets embedded in the documentation),
+and it is truly important to write the tests to ensure that the code works as intended.
+
+The package template provides a minimal setup and the tests can be executed by invoking the Pytest runner:
+
+```shell
+pytest
+```
+
+The runner looks for tests in `src` and `tests` folders. On top of that, the Python code
+in the top-level `README.md` file is also executed, so we can demonstrate
+the most imporant functionality:
+
+```python
+>>> from project_name.foo import foo
+>>> foo()
+True
+
+```
+
+Since the documentation is executed, any code changes that can render the documentation obsolete can be picked up,
+prompting the documentation update.
```

Comment:
The diff looks weird here, but only changed the word can to will.
```suggestion
Since the documentation is executed, any code changes that render the documentation obsolete will be picked up, forcing a documentation update.
```

Reply:
Thank you!